demo/deploy-MultiClusterService.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -auexo pipefail
+
+NS=bookstore
+NAME=bookstore
+
+kubectl delete MultiClusterService -n $NS $NAME || true
+
+kubectl apply -f - <<EOF
+apiVersion: config.openservicemesh.io/v1alpha1
+kind: MultiClusterService
+
+metadata:
+  namespace: $NS
+  name: $NAME
+
+spec:
+  clusters:
+  - name: alpha
+    # certificate
+    address: 1.99.99.1:7777
```

Comment:
Let's add a comment line, asking people to update this to a working address when they use it

Reply:
Added comments!